fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -764,7 +772,23 @@ private void handleReplicaVersionIncomplete(TabletSchedCtx tabletCtx, AgentBatch
             throws SchedException {
         stat.counterReplicaVersionMissingErr.incrementAndGet();
         try {
-            tabletCtx.chooseDestReplicaForVersionIncomplete(backendsWorkingSlots);
+            Set<Long> bes = Sets.newHashSet();
```

Comment:
bool skipAlwaysCloneFail = Config.create_new_replica_in_health_backends && backends.stream().anyMatch(be -> be.isSchedAvailable && backend not in tablet.backends && backend.getTag() == tablet.getTag() && backend contains disk tablet's storage medium
);